%% BioMed_Central_Tex_Template_v1.06
%%                                      %
%  bmc_article.tex            ver: 1.06 %
%                                       %

%%IMPORTANT: do not delete the first line of this template
%%It must be present to enable the BMC Submission system to
%%recognise this template!!

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                     %%
%%  LaTeX template for BioMed Central  %%
%%     journal article submissions     %%
%%                                     %%
%%          <8 June 2012>              %%
%%                                     %%
%%                                     %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                                                 %%
%% For instructions on how to fill out this Tex template           %%
%% document please refer to Readme.html and the instructions for   %%
%% authors page on the biomed central website                      %%
%% http://www.biomedcentral.com/info/authors/                      %%
%%                                                                 %%
%% Please do not use \input{...} to include other tex files.       %%
%% Submit your LaTeX manuscript as one .tex document.              %%
%%                                                                 %%
%% All additional figures and files should be attached             %%
%% separately and not embedded in the \TeX\ document itself.       %%
%%                                                                 %%
%% BioMed Central currently use the MikTex distribution of         %%
%% TeX for Windows) of TeX and LaTeX.  This is available from      %%
%% http://www.miktex.org                                           %%
%%                                                                 %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%% additional documentclass options:
%  [doublespacing]
%  [linenumbers]   - put the line numbers on margins

%%% loading packages, author definitions

\documentclass[twocolumn]{bmcart}% uncomment this for twocolumn layout and comment line below
%\documentclass{bmcart}

%%% Load packages
%\usepackage{amsthm,amsmath}
%\RequirePackage{natbib}
%\RequirePackage{hyperref}
\usepackage[utf8]{inputenc} %unicode support
%\usepackage[applemac]{inputenc} %applemac support if unicode package fails
%\usepackage[latin1]{inputenc} %UNIX support if unicode package fails


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                             %%
%%  If you wish to display your graphics for   %%
%%  your own use using includegraphic or       %%
%%  includegraphics, then comment out the      %%
%%  following two lines of code.               %%
%%  NB: These line *must* be included when     %%
%%  submitting to BMC.                         %%
%%  All figure files must be submitted as      %%
%%  separate graphics through the BMC          %%
%%  submission process, not included in the    %%
%%  submitted article.                         %%
%%                                             %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\def\includegraphic{}
\def\includegraphics{}



%%% Put your definitions there:
\startlocaldefs
\endlocaldefs


%%% Begin ...
\begin{document}

%%% Start of article front matter
\begin{frontmatter}

\begin{fmbox}
\dochead{Meeting Report}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the title of your article here     %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\title{Genomics by the beach}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the authors here                   %%
%%                                          %%
%% Specify information, if available,       %%
%% in the form:                             %%
%%   <key>={<id1>,<id2>}                    %%
%%   <key>=                                 %%
%% Comment or delete the keys which are     %%
%% not used. Repeat \author command as much %%
%% as required.                             %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\author[
   addressref={aff1, aff2},                   % id's of addresses, e.g. {aff1,aff2}
   noteref={n1},                        % id's of article notes, if any
   email={hickey@wehi.edu.au}   % email address
]{\inits{PFH}\fnm{Peter F.} \snm{Hickey}}
\author[
   addressref={aff3,aff4},
  noteref={n1},
   corref={aff3},                       % id of corresponding address, if any
   email={mark.robinson@imls.uzh.ch}
]{\inits{MDR}\fnm{Mark D.} \snm{Robinson}}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the authors' addresses here        %%
%%                                          %%
%% Repeat \address commands as much as      %%
%% required.                                %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\address[id=aff1]{%                           % unique id
  \orgname{Bioinformatics Division, Walter and Eliza Hall Institute of Medical Research}, % university, etc
  \street{1G Royal Parade},                     %
  \postcode{3052}                                % post or zip code
  \city{Parkville},                              % city
  \state{Victoria},
  \cny{Australia}                                    % country
}
\address[id=aff2]{%                           % unique id
  \orgname{Department of Mathematics and Statistics, University of Melbourne}, % university, etc
%  \street{Waterloo Road},                     %
  \postcode{3010}                                % post or zip code
  \city{Parkville},                              % city
  \state{Victoria},
  \cny{Australia}                                    % country
}
\address[id=aff3]{%
  \orgname{Institute of Molecular Life Sciences, University of Zurich},
  \street{Winterthurerstrasse 190},
  \postcode{8057}
  \city{Zurich},
  \cny{Switzerland}
}
\address[id=aff4]{%
  \orgname{SIB Swiss Insitute of Bioinformatics, University of Zurich},
  %\street{Winterthurerstrasse 190},
  %\postcode{8057}
  \city{Zurich},
  \cny{Switzerland}
}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter short notes here                   %%
%%                                          %%
%% Short notes will be after addresses      %%
%% on first page.                           %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{artnotes}
%\note{Sample of title note}     % note to the article
\note[id=n1]{Equal contributor} % note, connected to author
\end{artnotes}

%\end{fmbox}% comment this for two column layout

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% The Abstract begins here                 %%
%%                                          %%
%% Please refer to the Instructions for     %%
%% authors on http://www.biomedcentral.com  %%
%% and include the section headings         %%
%% accordingly for your article type.       %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{abstractbox}

\begin{abstract} % abstract
A report on the $35^{th}$ Annual Lorne Genome Conference 2014 held in Lorne, Victoria, Australia, February 16-18, 2014. 
\end{abstract}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% The keywords begin here                  %%
%%                                          %%
%% Put each keyword in separate \kwd{}.     %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{keyword}
\kwd{meeting report}
\kwd{Lorne Genome}
\kwd{author}
\end{keyword}

% MSC classifications codes, if any
%\begin{keyword}[class=AMS]
%\kwd[Primary ]{}
%\kwd{}
%\kwd[; secondary ]{}
%\end{keyword}

\end{abstractbox}
%
\end{fmbox}% uncomment this for twcolumn layout

\end{frontmatter}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% The Main Body begins here                %%
%%                                          %%
%% Please refer to the instructions for     %%
%% authors on:                              %%
%% http://www.biomedcentral.com/info/authors%%
%% and include the section headings         %%
%% accordingly for your article type.       %%
%%                                          %%
%% See the Results and Discussion section   %%
%% for details on how to create sub-sections%%
%%                                          %%
%% use \cite{...} to cite references        %%
%%  \cite{koon} and                         %%
%%  \cite{oreg,khar,zvai,xjon,schn,pond}    %%
%%  \nocite{smith,marg,hunn,advi,koha,mouse}%%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%% start of article main body
% <put your article body there>

%%%%%%%%%%%%%%%%
%% Background %%
%%

The $35^{th}$ Annual Lorne Genome Conference attracted scientists from around Australia and internationally to the Victorian Surf Coast town of Lorne for 3 days of genomics by the beach. The diversity of genomics research was on show across the 35 talks, including a session dedicated to young researchers and students, and more than 250 poster presentations.

The standard of scientific talks was extremely high.  However, in summarising the meeting, we have chosen to focus on presentations that i) we can most directly relate to; ii) that we found most interesting; and, iii) that we were able to find the best notes for \textbf{not sure if we want to say this last part.}.  Notably, only 15 of 35 talks were from Australian (or New Zealand) speakers, highlighting the variety of genomics talent, both in delegates and speakers, that remains the staple of the Lorne Genome meeting.  The setting in summertime, 40 metres from a very popular beach, does little to dampen the spirits of the speakers.


%a small number of talks the  \textbf{TODO: In summarising the meeting, we have chosen to focus on presentations $\ldots$}
%
%\textbf{TODO: I'm rather aware that most of the talks I've discussed are from "big name" international speakers - not great. But these are the talks (a) I found most interesting, (b) have the best notes on so can fairly summarise and (c) only 15/35 talks were given by people working in Aus/NZ, of which 4 were student prize talks, so most were international speakers in any case.}

\section*{Genomes Down Under}
Scientists have long been fascinated by Australia's unique wildlife. In more recent times, scientists have studied the genomes of Australia's marsupials and monotremes, such as the tammar wallaby and the platypus, to learn about embryonic development, sexual reproduction and the evolution of mammalian sex chromosomes. Genomics research has also been hugely important in the race to protect the Tasmanian devil from extinction in the wild due to the Tasmanian devil facial tumour disease. Continuing in this tradition, Katherine Belov (University of Sydney, Australia) spoke about the antimicrobial and venom peptides of Australian mammals. The male platypus is one of the few venomous mammals (but only one amongst many of Australia's other venomous animals). Belov is studying the transcriptome and proteome of platypus venom to learn about its evolutionarily origins. Belov also spoke about antimicrobial peptide genes found in marsupial and monotreme genomes. Belov presented evidence that these peptides are promising candidates as novel antibiotics against multi-drug resistant bacterial strains.  Will humans drink marsupial milk to ward off multi-resistant bugs ?


With the world-renowned Jenny Graves in the audience, Henrik Kasessmann (Centre for Integrative Genomics, University of Lausanne) gave a compelling insights into the evolution of Y chromosome and mammalian tissue transcriptomes.  Accurate chromosome Y sequence is already hard to come by, MORE HERE.
\textbf{Mark: Did you see Henrik Kasessmann's talk on "The Evolution of Mammalian Tissue Transcriptomes"? It would seem a good follow-on from Kathy's talk but unfortunately I missed it}

\section*{Disease and Medical Genomics(?)}
Elaine Mardis (The Genome Institute at Washington University, USA) gave a wonderful keynote presentation on translating cancer genomics research into clinical care. Mardis described how her team integrate whole genome sequencing (WGS), whole exome sequencing (WES) and RNA sequencing (RNA-seq). She emphasised the extra information that RNA-seq provides, as they find that only $40\%$ of mutated genes are expressed in the tumour (\textbf{Is this what she meant by the power of RNA-seq? In what context - Cancer generally or was she talking about a specific cancer at this point?}). Aside from the challenges of integrating multiple data sources, Mardis also described how these results are then presented to the Washington University School of Medicine's Tumour Board to help determine the clinical care of the patients. It is critical that physicians are well-trained in understanding the strengths of limitations of genomics research, Mardis told the audience, and described 3 (\textbf{or 4?}) cases presented to the review board. Not only were these fascinating examples of translating cancer genomics into clinical practice, but they also drew shocked laughs from the mostly-Australian audience about the realities of the American health care system. 

\textbf{TODO: Another presentation in this theme? Or give the section a new title and include some other presentations?}



\section*{Epigenomics and Genome Organisation}
Epigenomics and the higher-order organisation of the genome were the theme of several presentations. Bing Ren (University of San Diego School of Medicine, USA) spoke about detecting enhancers and re-constructing the 3-dimensional structure of the genome using the chromatin-conformation-capture technology, Hi-C. Ren showed that Hi-C has more strings to its proverbial bow and described his group's recent foray into using Hi-C for long-range haplotyping of human and mouse genomes. Resolving haplotypes is still a difficult problem, particularly in mammalian genetics where most sequencing is done using Illumina's short read technology. Haplotyping by Hi-C, along with recent improvements in throughput by Pacific Biosciences' longer read technology and Illumina's commercialisation of the Moleculo technology, are helping to "close the gap".

Wendy Bickmore (University of Edinburgh, UK) presented data showing that the most active regions of genomes are often found in the centre of the nucleus whereas the nuclear periphery is occupied by gene-poor, inactive chromosomal domains. Bickmore described recent experiments that seek to determine whether it is the genomic structure that dictates function or the function that dictates the structure by perturbing either the nuclear organisation or transcription. \textbf{TODO: Bit more on Wendy's talk.}

Joseph Ecker (Salk Institute, USA) closed the conference with a talk that took us from the world of human genomes to the that of plants' and back again through the common theme of DNA methylation. Ecker's lab published some of the first genome-wide maps of DNA methylation in human cells and continue to do so as part of the NIH Epigenomics Roadmap consortium. It is well known that DNA methylation in mammalian cells mostly occurs at CpG dinucleotides.  In contrast, non-CpG methylation is prevalent in plants, such as in the model organism \emph{Arabidopsis Thaliana}. Ecker presented work from his group and others that showed non-CpG methylation also exists in mammalian cells and can be widespread, particularly in pluripotent cell lines and in neurons. Ecker highlighted the power of model organisms by studying epialleles in inbred lines of \emph{A. Thaliana}. Epialleles are genetically identical sequences with stable methylation patterns that can result in phenotypic variation. The effect of epialleles, even their existence, are difficult to study in humans due to genetic variability. However, by careful breeding of \emph{A. Thaliana} strains,  Ecker's team were able to remove the genetic variation and focus on the phenotypic variation driven by differences in DNA methylation patterns.

\section*{Conclusions}
\textbf{TODO}

For an Australian government without a science minister, we should have a 'blokes' question: why is that the 

\begin{itemize}
 \item $43\%$ of talks by women - great!
 \item The females also dominated the poster prize with 5/6 (I think, perhaps even 5/5) winners (although as Alicia pointed, the picture is terrible for the Young Investigators Award which has been won by males all 15/15 times)
 \item Award winners?
 \item Thank the WEHI for supporting the Parent's Viewing room.
\end{itemize}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Backmatter begins here                   %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{backmatter}

\section*{Abbreviations}
WGS: whole genome sequencing; WES: whole exome sequencing; RNA-seq: RNA sequencing

\section*{Competing interests}
  The authors declare that they have no competing interests.

\section*{Author's contributions}
    Text for this section \ldots

\section*{Acknowledgements}
 Thanks to the Lorne Genome 2014 tweep superstars, most notably Christine Wells (@mincle), Thomas Preiss (@tominaustralia), Matthew Wakefield (@genomematt), Alicia Oshlack (@AliciaOshlack) and Jeff Craig (@DrChromo).

\section*{Twitter}
   The tweets from this conference can be found using the hashtag \#lornegenome. You can follow both authors on Twitter at @PeteHaitch and @markrobinsonca.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                  The Bibliography                       %%
%%                                                         %%
%%  Bmc_mathpys.bst  will be used to                       %%
%%  create a .BBL file for submission.                     %%
%%  After submission of the .TEX file,                     %%
%%  you will be prompted to submit your .BBL file.         %%
%%                                                         %%
%%                                                         %%
%%  Note that the displayed Bibliography will not          %%
%%  necessarily be rendered by Latex exactly as specified  %%
%%  in the online Instructions for Authors.                %%
%%                                                         %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% if your bibliography is in bibtex format, use those commands:
%\bibliographystyle{bmc-mathphys} % Style BST file
%\bibliography{bmc_article}      % Bibliography file (usually '*.bib' )

% or include bibliography directly:
% \begin{thebibliography}
% \bibitem{b1}
% \end{thebibliography}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                               %%
%% Figures                       %%
%%                               %%
%% NB: this is for captions and  %%
%% Titles. All graphics must be  %%
%% submitted separately and NOT  %%
%% included in the Tex document  %%
%%                               %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%
%% Do not use \listoffigures as most will included as separate files

%\section*{Figures}
%\begin{figure}[h!]
%\caption{\csentence{Sample figure title.}
%      A short description of the figure content
%      should go here.}
%      \end{figure}
%
%\begin{figure}[h!]
%  \caption{\csentence{Sample figure title.}
%      Figure legend text.}
%      \end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                               %%
%% Tables                        %%
%%                               %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%% Use of \listoftables is discouraged.
%%
%\section*{Tables}
%\begin{table}[h!]
%\caption{Sample table title. This is where the description of the table should go.}
%      \begin{tabular}{cccc}
%        \hline
%           & B1  &B2   & B3\\ \hline
%        A1 & 0.1 & 0.2 & 0.3\\
%        A2 & ... & ..  & .\\
%        A3 & ..  & .   & .\\ \hline
%      \end{tabular}
%\end{table}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                               %%
%% Additional Files              %%
%%                               %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%\section*{Additional Files}
%  \subsection*{Additional file 1 --- Sample additional file title}
%    Additional file descriptions text (including details of how to
%    view the file, if it is in a non-standard format or the file extension).  This might
%    refer to a multi-page table or a figure.
%
%  \subsection*{Additional file 2 --- Sample additional file title}
%    Additional file descriptions text.


\end{backmatter}
\end{document}
